pages/docs/community/styleguide/content-buckets.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Please remove mentions of _"this area needs to be concise/accurate/etc"_ because those descriptors apply to all documentation, regardless of the content bucket.